earth_enterprise/rpms/build.gradle

```diff
@@ -9,123 +9,265 @@ buildscript {
-
-apply plugin: 'nebula.ospackage'
+def openGeeVersion =
+    new File(project.projectDir, '../src/fusion_version.txt').readLines().
+    collect {
+        // Strip comments:
+        def commentIndex = it.indexOf('#')
+
+        commentIndex < 0 ? it : it.substring(0, commentIndex)        
+    }.
+    collect { it.trim() }. // Trim whitespace.
+    findAll { !it.isEmpty() }. // Skip empty lines.
+    findResult {
+        it // The first string that remains is the version string.
+    }
+
+
+def stagedInstallDir = new File('/tmp/fusion_os_install')
+def stagedInstallDir_common = new File(stagedInstallDir, 'common')
+def stagedInstallDir_common_opt = new File(stagedInstallDir, 'common/opt')
+def stagedInstallDir_common_user_magic =
+    new File(stagedInstallDir, 'common/user_magic')
+def stagedInstallDir_manual = new File(stagedInstallDir, 'manual')
+def stagedInstallDir_server_opt = new File(stagedInstallDir, 'server/opt')
+def stagedInstallDir_server_etc = new File(stagedInstallDir, 'server/etc')
+def stagedInstallDir_server_apachesupport =
+    new File(stagedInstallDir, 'server/AppacheSupport/')
+def stagedInstallDir_server_usermagic =
+    new File(stagedInstallDir, 'server/user_magic/')
+def stagedInstallDir_fusion = new File(stagedInstallDir, 'fusion')
+def packageInstallRootDir = new File('/')
+
+def postGisInstallDir = new File(
+        project.projectDir,
+        '../src/NATIVE-REL-x86_64/third_party/postgis/install'
+    ).getAbsolutePath()
+
+def postGisInstallDir_opt = new File(postGisInstallDir, 'opt')
+
 
 ospackage {
-    version = packageVersion
-    release = packageRelease
+    version = openGeeVersion
+    release = '1'
     arch = X86_64
     os = LINUX
-    into '/opt/google'
 }
 
-task commonRpm (type: Rpm) {
+task compileOpenGee(type: Exec) {
+    workingDir = '..'
+
+    commandLine('scons', '-j8', 'release=1', 'build')
+}
+
+task stageOpenGeeInstall(type: Exec, dependsOn: 'compileOpenGee') {
+    workingDir = '..'
+
+    // The scons `stage_install` task doesn't work correctly with
+    // "installdir=${build_install_dir.getAbsolutePath()}", so we
+    // overwrite whatever is in the default stage install directory:
+    commandLine(
+        'scons', '-j8', 'release=1', 'stage_install')
+}
+
+task openGeePostGisRpm(type: Rpm, dependsOn: 'stageOpenGeeInstall') {
+    packageName = 'opengee-postgis'
+    release = '1'
+    version = '1.5.8'
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+    summary = 'Post GIS extension for Postgre SQL server used by Open GEE'
+    packageDescription = 'Post GIS extension for Postgre SQL server used by Open GEE.'
+    license = 'GPL 2'
+    packager = 'gee-oss@googlegroups.com'
+    url = 'https://github.com/google/earthenterprise/tree/master/earth_enterprise/third_party/postgis'
+    arch = 'X86_64'
+    os = 'LINUX'
+    type = 'BINARY'
+
+    from (postGisInstallDir_opt) {
```

Comment:
Any reason PostGIS isn't included in the same RPM as the other 3rd party packages.

Reply:
It has a GPL license, which usually implies that derived works have to be licensed under GPL, as well.  We're packaging separately, so as not to produce a package that's a derived work.